src/main/java/org/apache/sling/maven/slingstart/PackageMojo.java

```diff
@@ -51,6 +52,13 @@
      */
     @Parameter(defaultValue="false")
     protected boolean createWebapp;
+    
+    /**
+     * If set to {@code true} does not attach the generated artifact to Maven.
```

Comment:
I think you need to change the JavaDoc here with `{@code false}`, right?

Reply:
Good catch, fixed with https://github.com/apache/sling-slingstart-maven-plugin/commit/d6203e5aaadc1ad10bc11625bf7f8f1d92299da3.